src/app/helptext/system/email.ts

```diff
@@ -1,10 +1,11 @@
 import { Validators } from "@angular/forms";
 import { T } from "app/translate-marker";
-import { matchOtherValidator } from "app/pages/common/entity/entity-form/validators/password-validation";
+import { rangeValidator } from 'app/pages/common/entity/entity-form/validators/range-validation'
 
 export const helptext_system_email = {
   fromemail: {
     placeholder: T("From E-mail"),
+    validation: [Validators.required],
```

Comment:
You could also use Validators.email here to prevent invalid email addresses being typed in